src/libraries/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Configurations>$(NetCoreAppCurrent)-Debug;$(NetCoreAppCurrent)-Release;$(NetFrameworkCurrent)-Debug;$(NetFrameworkCurrent)-Release</Configurations>
+    <UnicodeUcdVersion>12.1</UnicodeUcdVersion>
```

Comment:
A lot of projects define this property now and that makes it hard to keep on sync. I think it would be better to add a .props file to the nuget package and define the property there, then it will automatically be imported to the consuming projects and available.

Reply:
Followed up offline. The intent is that the _runtime-assets_ package can contain multiple different versions and that consuming projects should be able to specify exactly which version of the data files they care about. For a long time _System.Runtime_ was using version 11.0 and _System.Text.Encodings.Web_ was using version 12.1.

We were brainstorming ideas offline about how to resolve this, such as having a property per version of the package or having a global default that individual projects can override as necessary. We'll continue to have those discussions offline and won't hold up the PR for this right now.